wiki/Application-Issues.md

```diff
@@ -30,3 +30,19 @@ To fix this, run them with the `_JAVA_AWT_WM_NONREPARENTING=1` environment varia
 
 There's a bug in rofi-wayland that prevents it from accepting keyboard input on niri with errors in the output.
 It's been fixed in rofi, but the fix had not been released yet: https://github.com/davatorium/rofi/discussions/2008
+
+### Fullscreen games
+
+Many video games, both Linux-native and on Wine, can have all sorts of issues when using non-stacking desktop environments.
```

Comment:
I'd downgrade this at least to "Some". I've played through several big and small games on niri through xwayland-satellite, and haven't really been hitting many problems.